src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -1761,30 +1761,21 @@ public void loadGestures() {
    * @param directory
    *          - the directory that contains the gesture python files.
    */
-  public void loadGestures(String directory) {
-    // TODO: iterate over each of the python files in the directory
+  public boolean loadGestures(String directory) {
+
+    // iterate over each of the python files in the directory
     // and load them into the python interpreter.
     File dir = makeGesturesDirectory(directory);
+    boolean loaded = true;
     for (File f : dir.listFiles()) {
-      if (f.getName().toLowerCase().endsWith(".py")) {
-        log.info("Loading Gesture Python file {}", f.getAbsolutePath());
-        Python p = (Python) Runtime.getService("python");
-        String script = null;
-        try {
-          script = FileIO.toString(f.getAbsolutePath());
-        } catch (IOException e) {
-          log.warn("IO Error loading gesture file {} -- {}", f.getAbsolutePath(), e);
-          continue;
-        }
-        // evaluate the gestures scripts in a blocking way.
-        boolean result = p.exec(script, true, true);
-        if (!result) {
-          log.warn("Error while loading gesture file: {}", f.getAbsolutePath());
-        } else {
-          log.info("Successfully loaded gesture {}", f.getAbsolutePath());
-        }
-      }
+      loaded = Utils.loadPythonFile(f.getAbsolutePath());
     }
+    return true;
```

Comment:
You are also missing the point that this method previously would skip failed scripts.  This is a highly desirable behavior because it means it will continue to load gestures even if one of them has a syntax error in it.